### ISSUE TITLE
About the Results of the experiment

### ISSUE THREAD

**Issue by CiChina** [mydcxiao/ReferSAM#1]
Regarding your test results on the refcoco dataset proposed by readme, can you tell me what your training conditions were? Like loaded pre-trained models etc. I find that the accuracy of my own trained models falls far short of the expected results.

**Comment by mydcxiao** [comment]
As far as I remember, the best results came from finetuning the pretrained SAM weights. The trainable parameters are the text prompt encoder and decoder leaving the ViT frozen. You can try lambdaLR and CosineAnnealLR to see which one performs better.

**Comment by CiChina** [comment]
thank you for your good job

**Comment by mydcxiao** [comment]
> thank you for your good job

Thanks! Let's close the issue.
